src/halmos/__main__.py

```diff
@@ -790,7 +810,7 @@ class RunArgs:
     methodIdentifiers: Dict[str, str]
 
 
-def run_parallel(run_args: RunArgs) -> Tuple[int, int]:
+def run_parallel(run_args: RunArgs) -> Tuple[int, int, List[TestResult]]:
```

Comment:
it would feel cleaner to have the following signatures instead:

```
def run_parallel(run_args: RunArgs) -> List[TestResult]:
def run_sequential(run_args: RunArgs) -> List[TestResult]:
```

and the consumer can easily derive `num_passed` and `num_failed` from the `List[TestResult]`

Reply:
good idea! done.